### PR TITLE
Development

### PR DESCRIPTION
**Pull request by SotoArmando**
First code review.

### CODE REVIEW

**Review by inhaq — CHANGES_REQUESTED** [review]
Good work on your first project 🏆
Here are a few things to fix and improve:

 - Fix your errors by validating it [here](https://validator.w3.org)
 - Improve design by looking it more like YouTube. You can improve search bar, add logo, text to sidebar etc.
 - Add ReadMe file to the repository to add details about your project.

Once fixed, kindly resubmit and add me(Inzi) as project reviewer. Happy Coding 😊

**Comment by Ispirett** [comment]
Thanks for your feed back will do

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+  background-color: rgba(255, 255, 255, 0.15);
+  border-radius: 0.25rem;
+  border-bottom-left-radius: 0;
+  border-top-left-radius: 0;
+  margin-left: -5px;
+}
+
+#search-bar {
+  height: 1.75rem;
+  text-align: center;
```
Comment (inhaq) [inline css/main.css:99]: `text-align: center;` need to be removed to make it look better

**Review by inhaq — APPROVED** [review]
### Amazing work on your first project 🏆
I really liked dark-mode navbar, it's designed very clean.
Everything looks good. I would also recommend to not use `id` in your next projects.
Go ahead and submit project completion form and no need to submit for review again this project. Happy Coding 😊

**Pull request merged**